src/main/java/com/example/bookstore/controller/admin/OrdersController.java

```diff
@@ -35,13 +43,44 @@ public String index(Model model,
         model.addAttribute("orders", orders);
 
         int totalPages = orders.getTotalPages();
-        if (totalPages > 0) {
-            List<Integer> pageNumbers = IntStream.rangeClosed(1, totalPages)
-                    .boxed()
-                    .collect(Collectors.toList());
-            model.addAttribute("pageNumbers", pageNumbers);
+        if (totalPages <= 0) {
+            return "admin/orders/index";
         }
+        List<Integer> pageNumbers = IntStream.rangeClosed(1, totalPages)
+                .boxed()
+                .collect(Collectors.toList());
+        model.addAttribute("pageNumbers", pageNumbers);
 
         return "admin/orders/index";
     }
-}
+
+    @GetMapping("/{id}/details")
```

Comment:
```suggestion
    @GetMapping("/{id}")
```
ham hố để bị comment =))

Reply:
Để em sửa ạ